app/models/govuk_publishing_components/component_fixture.rb

```diff
@@ -0,0 +1,15 @@
+module GovukPublishingComponents
+  ComponentFixture = Struct.new(:id, :data) do
+    def name
+      id.humanize
+    end
+
+    def pretty_data
+      JSON.pretty_generate(data).gsub('\\n', "\n    ").gsub(/"(\w*)":/, '\1:').strip
```

Comment:
When performing this sort of `gsub` wizardry, it's often nice to break it out into a one-line mutating method with a nice name so that people can understand what's going on, eg:

```ruby
def pretty_data
  prettified = JSON.pretty_generate(data)
  prepend_new_lines_with_whitespace!(prettified)
  unquote_properties!(prettified)
  prettified
end
```

On that note, just why _are_ we doing these things? To make a Ruby-looking hash? In that case maybe we should be using a gem? Or at least rename this method to something like `data_as_pretty_printed_ruby_hash`?